server/ai_live_video.go

```diff
@@ -199,30 +198,47 @@ func startEventsSubscribe(ctx context.Context, url *url.URL, params aiRequestPar
 				continue
 			}
 
-			stream := params.liveParams.stream
-
-			if stream == "" {
-				clog.Infof(ctx, "Stream ID is missing")
-				continue
-			}
-
-			var status monitor.PipelineStatus
+			var status map[string]interface{}
 			if err := json.Unmarshal(body, &status); err != nil {
 				clog.Infof(ctx, "Failed to parse JSON from events subscription: %s", err)
 				continue
 			}
 
-			status.StreamID = &stream
+			if _, ok := status["stream_id"]; !ok {
+				clog.Infof(ctx, "Received event for stream=%s status=%+v", stream, status)
+				status["stream_id"] = stream
+				status["request_id"] = params.liveParams.requestID
+			}
 
-			// TODO: update the in-memory pipeline status
-			// monitor.UpdatePipelineStatus(stream, status)
+			monitor.SendQueueEventAsync("stream_status", status)
```

Comment:
Similarly, this should also not be called `stream_status`. Maybe `stream_events`? Or even `ai_stream_events`? (or in the singular form, not sure what's the kafka pattern rn)

Reply:
This is a requirement from data structure. It's a specific event type for this kind of data - other events may have different naming.
cc @ecmulli for input

Reply:
I'm assuming this is related to our discord [conversation](https://discord.com/channels/423160867534929930/1316106387666894888/1316419426286829580).

`stream_status` refers to the interval checks that outputs fps, restarts, etc. I'd also accept `ai_stream_status` if its important to distinguish between ai and regular video.

lets use `stream_events` (or `ai_stream_events`) to capture the actual events, i.e. errors/restarts/params_update.

Reply:
The higher level context is that these names are used in clickhouse to create specific tables. for example `type=network_payment` gets directed into a payments table. but they all get ingested via the same kafka topic
<img width="358" alt="image" src="https://github.com/user-attachments/assets/4efabfdf-9488-4f44-afcc-77a64166b40e" />